prow/integ-suite-kind.sh

```diff
@@ -37,10 +37,11 @@ setup_and_export_git_sha
 source "${ROOT}/common/scripts/kind_provisioner.sh"
 
 TOPOLOGY=SINGLE_CLUSTER
-NODE_IMAGE="gcr.io/istio-testing/kind-node:v1.23.0"
+NODE_IMAGE="gcr.io/istio-testing/kind-node:v1.24.0-0.13.0"
 KIND_CONFIG=""
 CLUSTER_TOPOLOGY_CONFIG_FILE="${ROOT}/prow/config/topology/multicluster.json"
-
+go install sigs.k8s.io/kind@v0.13.0
+kind version
```

Comment:
this was just for testing, we should not merge. And should instead upstream into build-tools

Reply:
will send a PR

Reply:
PTAL